### PR TITLE
fix typos in documentation

### PR DESCRIPTION
**Pull request by nitram509**
while I was briefly reading the documentation I found and fixed some typos.

I hope this helps a bit.

Any feedback welcome.

### CODE REVIEW

File: doc/CX10Blue.txt [hunk 1]
```diff
@@ -131,7 +131,7 @@ XOR with a fixed whitening sequence:
 Packets are GFSK modulated at a rate of 1 Mbit/second and a frequency deviation
 of approximately +/- 340 kHz.  Zero bits are represented by the lower frequency
 deviation while one bits are represented by the higher deviation.  Each packet
-has a duration of 245 microseconds and is preceded by an modulated carrier
+has a duration of 245 microseconds and is preceded by an not modulated carrier
```
Comment (tracernz) [inline doc/CX10Blue.txt]: _a_ not modulated... it made a lot more sense before (an unmodulated)...

Comment (goebish) [reply doc/CX10Blue.txt]: I'm not an English native, but unmodulated sounds fine to me.

Comment (mdmbc) [reply doc/CX10Blue.txt]: Indeed, unmodulated is fine.

**Comment by nitram509** [comment]
thank you for the feedback, gentlemen.
I did update the wording.

**Pull request merged**